app/src/main/java/net/squanchy/tweets/service/TweetModelConverter.java

```diff
@@ -120,6 +114,25 @@ private int offsetFrom(int start, List<Integer> indices) {
         return Lists.map(photos, mediaEntity -> mediaEntity.mediaUrlHttps);
     }
 
+    private String displayableTextFor(Tweet tweet, Range displayTextRange, List<MediaEntity> photoUrls) {
+        Integer beginIndex = displayTextRange.start();
+        Integer endIndex = displayTextRange.end();
+        String displayableText = tweet.text.substring(beginIndex, endIndex);
+        if (photoUrls != null) {
```

Comment:
⚠️  This should NEVER be null as it's not `@Nullable` -- check where it's created, should be an empty list in case there's no photos.